data-otservbr-global/scripts/creaturescripts/others/bosstiary_kill.lua

```diff
@@ -4,14 +4,15 @@ function bosstiaryOnKill.onKill(player, creature, lastHit)
 		return true
 	end
 
+	local bosstiaryMultiplier = (configManager.getNumber(configKeys.BOSSTIARY_KILL_MULTIPLIER) or 1)
+	local killBonus = (configManager.getNumber(configKeys.BOOSTED_BOSS_KILL_BONUS) or 3)
```

Comment:
```suggestion
```

Reply:
I don't understand why I have to remove this.